package.json

```diff
@@ -31,5 +31,8 @@
     "stylelint-scss": "^4.1.0",
     "stylelint-selector-bem-pattern": "^2.1.1",
     "stylelint-suitcss": "^5.0.0"
+  },
+  "devDependencies": {
+    "stylelint-config-idiomatic-order": "^8.1.0"
```

Comment:
It should be in `"dependencies"`, otherwise it will not be added to the tool.

Reply:
Done!